amazonia/amz.py

```diff
@@ -63,11 +68,15 @@ def main():
     """
     template_file_path = args.template
     template_trop = create_stack(stack_input)
+    send_to_output = args.out
     template_data = template_trop.template.to_json(indent=2, separators=(',', ': '))
     with open(template_file_path, 'w') as template_file:
```

Comment:
Yeah, that was an oversight, thanks. New to python and not confident about how to correctly use `with` syntax. See change.